
CommandArmServoAngle[]   desried_arm_rad
